djpykafka/events/subscribe.py

```diff
@@ -158,18 +158,42 @@ def __init_subclass__(
             topic_use_regex,
         )
 
+    def parse_event(self) -> DataChangeEvent:
+        if self.body == 'null':  # Debezium Tombstone events
+            return DataChangeEvent(
+                data_op=DataChangeEvent.DataOperation.DELETE,
+                data_type='debezium_tombstone',
+            )
+
+        return super().parse_event()
+
     def parse_body(self):
         contents = json.loads(super().parse_body())
 
+        if not contents:
+            event_type = 'debezium_tombstone'
+            data = None
+            operation = DataChangeEvent.DataOperation.DELETE
+            timestamp = datetime.now()
```

Comment:
further improvement: the timestamp should then be retrieved from the kafka message if possible